src/msg/async/AsyncMessenger.cc

```diff
@@ -198,8 +198,7 @@ void Processor::accept()
 	  w, std::move(cli_socket),
 	  msgr->get_myaddrs().v[listen_socket.get_addr_slot()],
 	  addr);
-	accept_error_num = 0;
-	continue;
+	break;
```

Comment:
why? before this change, `Processor::accept()` accepts one or more client connections until `EAGAIN` is returned which implies that there is no more connection to be connected. but after this change, `accept()` stops after establishing the first connection.

Reply:
Yes you are right. I misunderstand the accept. Sorry for this.